lms/djangoapps/experiments/factories.py

```diff
@@ -10,7 +10,7 @@
 from lms.djangoapps.experiments.models import ExperimentData, ExperimentKeyValue
 
 
-class ExperimentDataFactory(factory.DjangoModelFactory):  # lint-amnesty, pylint: disable=missing-class-docstring
+class ExperimentDataFactory(factory.django.DjangoModelFactory):  # lint-amnesty, pylint: disable=missing-class-docstring
```

Comment:
It'd be nice to do all the attribute navigation in the import for these files also, if practical.